test/request_test.dart

```diff
@@ -334,4 +334,10 @@ void main() {
       expect(request.toString(), 'POST $dummyUrl');
     });
   });
+
+  group('#invalid method', () {
```

Comment:
The name of the group is the getter/method being tested, so just use `"#method"`.
Then name the test to describe the error, maybe `"must be a token"`.

Reply:
fixed. Thanks